doc/ext/sphinx_gallery/docs_resolv.py

```diff
@@ -0,0 +1,435 @@
+# -*- coding: utf-8 -*-
+# Author: Óscar Nájera
```

Comment:
Good idea. What would the best way to include such a reference? Add a README.txt file in the ``sphinx_gallery`` directory?